engine/src/main/java/com/redhat/cloud/notifications/processors/email/EmailProcessor.java

```diff
@@ -0,0 +1,108 @@
+package com.redhat.cloud.notifications.processors.email;
+
+import com.redhat.cloud.notifications.DelayedThrower;
+import com.redhat.cloud.notifications.db.repositories.EndpointRepository;
+import com.redhat.cloud.notifications.db.repositories.TemplateRepository;
+import com.redhat.cloud.notifications.events.EndpointProcessor;
+import com.redhat.cloud.notifications.models.Endpoint;
+import com.redhat.cloud.notifications.models.EndpointType;
+import com.redhat.cloud.notifications.models.Event;
+import com.redhat.cloud.notifications.models.InstantEmailTemplate;
+import com.redhat.cloud.notifications.processors.ConnectorSender;
+import com.redhat.cloud.notifications.processors.SystemEndpointTypeProcessor;
+import com.redhat.cloud.notifications.processors.email.connector.dto.EmailNotification;
+import com.redhat.cloud.notifications.processors.email.connector.dto.RecipientSettings;
+import com.redhat.cloud.notifications.templates.TemplateService;
+import io.quarkus.logging.Log;
+import io.quarkus.qute.TemplateInstance;
+import io.vertx.core.json.JsonObject;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+@ApplicationScoped
+public class EmailProcessor extends SystemEndpointTypeProcessor {
+    @Inject
+    ConnectorSender connectorSender;
+
+    @Inject
+    EndpointRepository endpointRepository;
+
+    @Inject
+    EmailSubscriptionTypeProcessor emailSubscriptionTypeProcessor;
+
+    @Inject
+    TemplateRepository templateRepository;
+
+    @Inject
+    TemplateService templateService;
+
+    @Override
+    public void process(final Event event, final List<Endpoint> endpoints) {
+        DelayedThrower.throwEventually(EndpointProcessor.DELAYED_EXCEPTION_MSG, accumulator -> {
```

Comment:
You were right about `DelayedThrower` not being needed in this class. Sorry for asking you to add it!

Reply:
No problem! It is an easy change!